src/confdb/confdb.h

```diff
@@ -309,7 +309,7 @@
 #define CONFDB_PC_TYPE_CERT_AUTH "cert_auth"
 
 struct confdb_ctx;
-struct config_file_ctx;
+struct file_watch_ctx;
```

Comment:
What is the purpose of this forward declaration here?

Reply:
It was already there an I just renamed it. But you are right in that it seems it is not needed any more. I'm removing it.